pkg/operator/cluster/ceph/mon/health_test.go

```diff
@@ -168,12 +168,14 @@ func TestCheckHealthTwoMonsOneNode(t *testing.T) {
 
 	// add two mons to the mapping on node0
 	c.mapping.Node["mon1"] = &NodeInfo{
-		Name:    "node0",
-		Address: "0.0.0.0",
+		Name:     "node0",
+		Hostname: "node0",
+		Address:  "0.0.0.0",
 	}
 	c.mapping.Node["mon2"] = &NodeInfo{
-		Name:    "node0",
-		Address: "0.0.0.0",
+		Name:     "node0",
+		Hostname: "node0",
```

Comment:
Can you please check with a hostname that differs from the name to also test that scenario?